### PR TITLE
Add CSS/SCSS linitng using stylelint

### PR DESCRIPTION
**Pull request by saranshdhingra**

### CODE REVIEW

File: web/.stylelintrc.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "stylelint-config-standard-scss",
+    "rules": {
+        "color-function-notation": "legacy",
+        "alpha-value-notation": "number",
+        "selector-class-pattern": [
+			"^([a-z][a-z0-9]*)(-[a-z0-9]+)*(__([a-z][a-z0-9]*)(-[a-z0-9]+)*)*$",
```
Comment (saranshdhingra) [inline web/.stylelintrc.json:7]: This is done to allow some default classes of `Material Design`.
It has the format: `classname-word2__after-separator`

File: web/package.json [hunk 2]
```diff
@@ -14,12 +14,14 @@
...
   },
   "devDependencies": {
-    "@parcel/transformer-sass": "^2.2.0",
+    "@parcel/transformer-sass": "2.2.0",
     "eslint-config-google": "0.14.0",
     "eslint-config-prettier": "8.3.0",
     "eslint-plugin-prettier": "4.0.0",
     "parcel": "^2.0.0",
-    "prettier": "2.4.1"
+    "prettier": "2.4.1",
```
Comment (enocom) [inline web/package.json:22]: Since we're linting things, can we add a check to `npm run lint` that makes sure all things have been prettified?

Comment (saranshdhingra) [reply web/package.json:22]: I had added the test in `workflows/tests.yml` but this is better. Thanks for mentioning that.

File: web/package.json [hunk 3]
```diff
@@ -10,16 +10,18 @@
   "scripts": {
     "start": "parcel",
     "build": "parcel build",
-    "lint": "eslint src/",
-    "fix": "prettier --write src/ && eslint --fix src/"
+    "lint": "eslint src/ && stylelint src/css",
```
Comment (enocom) [inline web/package.json:13]: There's actually a stylelint plugin for eslint:

https://github.com/stylelint/eslint-config-stylelint

Ditto with prettier:

https://github.com/prettier/prettier-eslint

How about consolidating all linting into eslint?

Comment (saranshdhingra) [reply web/package.json:13]: @enocom 
Trying https://github.com/stylelint/eslint-config-stylelint I found that it only adds rules to our current JS ruleset.
This doesn't add functionality to use stylelint(via eslint).
I maybe wrong, but I tried different possibilities and the only effect I achieved was that I was able to add some rules to our eslint ruleset.

The package does say:
> stylelint org's shareable config for ESLint.

So, probably it's just a config shared by the org for JS files??

Comment (enocom) [reply web/package.json:13]: Oh I misread. It's JS only. OK.

File: web/package.json (hunk 2 above)
Comment (enocom) [reply web/package.json:22]: I'd like to see us incorporate prettier into the `lint` run script. But that can wait for a future PR.

**Review by enocom — APPROVED** [review]

File: web/package.json (hunk 2 above)
Comment (saranshdhingra) [reply web/package.json:22]: Yeah, surely :)

**Pull request merged**